File: yahtzeeCompressed.lua
array = {0, math.random (1,1296)} --Random number & counter. By OOQQ
while (array[2] ~= 666) do --We are playing with the number 666, checks if we get lucky this time		Loop Starts
  array[1] = array[1] + 1 --Increases dice counter by 1
  array[2] = math.random (1,1296) --1 in 1296 random generator
  print(array[1], array[2]) --Display stats at true
end --Loop Ends

File: yahtzeeCompressed.js
var diceToss = [0,0]; // Counter // by OOQQ
let random = () => Math.floor(Math.random() * 1296 + 1); // 1 in 1296 random generator
do { diceToss[0] = random(); // Random number stored in array index 0 						-Loop Starts
	diceToss[1]++; // Increases dice counter by 1
} while (diceToss[0] != 666); // We are playing with the number 666, checks if we get lucky this time		-Loop Ends
console.log = "Yahtzee took " + diceToss[1] + " throws!"; // Displays stats at true
